tic-tac-toe/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
Please, rewrite readme according to your project

Reply:
I made this.